README.md

```diff
@@ -1,6 +1,7 @@
 # ImmutableRecord
 
-TODO: Write a gem description
+ImmutableRecord provides simple immutable data structures with accessor
+methods and ActiveRecord style mass-assignment initialization.
```

Comment:
still, it doesn't behave like AR mass assignment as you can omit attributes with AR, but here all arguments are required